src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from "react-router-dom";
+
+export default function ErrorPage({error}) {
+  return error ?  (
+    <h2>{error.data.msg}</h2>
+  ) : (
+    <div className="error-page">
+      <h2>Oh no, it looks like this page doesn't exist anymore.</h2>
```

Comment:
The 'anymore' bit might not always be true, the page might have never existed in the first place like when i navigated to '/sfsgokjsidnsa'